v3/src/components/graph/components/graph-inspector.tsx

```diff
@@ -42,12 +44,36 @@ export const GraphInspector = ({ tile, show }: ITileInspectorPanelProps) => {
     buttonRef.current = ref.current
   }
 
-  return (
-    <InspectorPanel ref={panelRef} component="graph" show={show} setShowPalette={setShowPalette}>
-      <InspectorButton tooltip={t("DG.Inspector.resize.toolTip")} showMoreOptions={false}
-        testId={"graph-resize-button"} onButtonClick={handleClosePalette}>
+  const renderRescaleButton = () => {
+
+    const getRescaleTooltip = () => {
+      if (graphModel?.noPossibleRescales) {
+        return "V3.Inspector.rescale.noRescale.toolTip"
+      }
+      else if (graphModel?.plotType === 'casePlot') {
+        return "V3.Inspector.rescale.casePlot.toolTip"
+      }
+      return "DG.Inspector.rescale.toolTip"
+    }
```

Comment:
I'm not sure how much this matters really, but it doesn't seem like a function is necessary for this. Wouldn't something like the below work just as well?

```
const getRescaleTooltip = graphModel?.noPossibleRescales
  ? "V3.Inspector.rescale.noRescale.toolTip"
  : graphModel?.plotType === "casePlot"
    ? "V3.Inspector.rescale.casePlot.toolTip"
    : "DG.Inspector.rescale.toolTip"
```